lib/SIL/GraphFunctionDeviceInfo.cpp

```diff
@@ -14,21 +14,69 @@
 //
 //===----------------------------------------------------------------------===//
 
-
 #include "swift/SIL/GraphFunctionDeviceInfo.h"
 #include "swift/AST/DiagnosticsSIL.h"
 #include "swift/SIL/GraphOperationBuilder.h"
 #include "swift/SIL/GraphOperationInfo.h"
-#include "swift/SIL/SILInstruction.h"
 #include "swift/SIL/SILFunction.h"
+#include "swift/SIL/SILInstruction.h"
 #include "swift/SILOptimizer/Utils/Local.h"
-#include "llvm/ADT/StringRef.h"
+#include "tensorflow/c/c_api_experimental.h"
 #include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
 #include "llvm/Support/CommandLine.h"
+#include <iostream>
+
```

Comment:
do you want to move this empty line above the previous line? and what do we need `<iostream>` for?